app/src/main/java/mod/hey/studios/project/proguard/ProguardHandler.java

```diff
@@ -176,6 +177,33 @@ public void setProguardEnabled(boolean proguardEnabled) {
         FileUtil.writeFile(config_path, new Gson().toJson(config));
     }
 
+    public boolean isR8Enabled() {
+        boolean r8Enabled = true;
+        if (FileUtil.isExistFile(config_path)) {
+            try {
+                HashMap<String, String> config = new Gson().fromJson(FileUtil.readFile(config_path), TypeToken.get(HashMap.class).getType());
```

Comment:
why didn't you use `Helper.TYPE_STRING_MAP` like the other methods? since replacing the explicit type before `config` with `var` makes `config` be of type `Object`, I don't think that approach works and is probably treated as a semi-unsafe cast